From e253b2dd34112b6ec2d7197249fb69e8bea9f88d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E5=BE=90=E6=8C=81=E6=81=92=20Xu=20Chiheng?=
 <chiheng.xu@gmail.com>
Date: Wed, 10 Apr 2024 04:18:03 +0800
Subject: [PATCH 20/39] managedbuilder.gnu.ui/plugin.xml : remove all
 envVarBuildPath elements and references to CygwinPathResolver

---
 .../plugin.xml                                | 44 -------------------
 1 file changed, 44 deletions(-)

diff --git a/build/org.eclipse.cdt.managedbuilder.gnu.ui/plugin.xml b/build/org.eclipse.cdt.managedbuilder.gnu.ui/plugin.xml
index ce11ac1764..4a857e4c41 100644
--- a/build/org.eclipse.cdt.managedbuilder.gnu.ui/plugin.xml
+++ b/build/org.eclipse.cdt.managedbuilder.gnu.ui/plugin.xml
@@ -24,10 +24,6 @@
     	        <value id="org.eclipse.cdt.build.core.buildArtefactType.sharedLib"/>
             </property>
             </supportedProperties>
-         <envVarBuildPath 
-         	pathType="buildpathLibrary"
-         	variableList="LIBRARY_PATH">
-         </envVarBuildPath>
          
          <optionCategory
                owner="cdt.managedbuild.tool.gnu.c.linker"
@@ -250,10 +246,6 @@
     	        <value id="org.eclipse.cdt.build.core.buildArtefactType.sharedLib"/>
             </property>
             </supportedProperties>
-         <envVarBuildPath 
-         	pathType="buildpathLibrary"
-         	variableList="LIBRARY_PATH">
-         </envVarBuildPath>
          <optionCategory
                owner="cdt.managedbuild.tool.gnu.cpp.linker"
                name="%OptionCategory.General"
@@ -628,10 +620,6 @@
     	        <value id="org.eclipse.cdt.build.core.buildArtefactType.sharedLib"/>
             </property>
             </supportedProperties>
-         <envVarBuildPath 
-         	pathType="buildpathLibrary"
-         	variableList="LIBRARY_PATH">
-         </envVarBuildPath>
          
          <optionCategory
                owner="cdt.managedbuild.tool.macosx.c.linker"
@@ -819,10 +807,6 @@
     	        <value id="org.eclipse.cdt.build.core.buildArtefactType.sharedLib"/>
             </property>
             </supportedProperties>
-         <envVarBuildPath 
-         	pathType="buildpathLibrary"
-         	variableList="LIBRARY_PATH">
-         </envVarBuildPath>
          <optionCategory
                owner="cdt.managedbuild.tool.macosx.cpp.linker"
                name="%OptionCategory.General"
@@ -985,10 +969,6 @@
     	        <value id="org.eclipse.cdt.build.core.buildType.release"/>
             </property>
             </supportedProperties>
-         <envVarBuildPath 
-         	pathType="buildpathInclude"
-         	variableList="CPATH,C_INCLUDE_PATH">
-         </envVarBuildPath>
 		  <optionCategory
 			  owner="cdt.managedbuild.tool.gnu.c.compiler"
 			  name="%OptionCategory.Dialect"
@@ -1534,10 +1514,6 @@
     	        <value id="org.eclipse.cdt.build.core.buildType.release"/>
             </property>
             </supportedProperties>
-         <envVarBuildPath 
-         	pathType="buildpathInclude"
-         	variableList="CPATH,CPLUS_INCLUDE_PATH">
-         </envVarBuildPath>
 		  <optionCategory
 			  owner="cdt.managedbuild.tool.gnu.cpp.compiler"
 			  name="%OptionCategory.Dialect"
@@ -2122,11 +2098,6 @@
           id="cdt.managedbuild.tool.gnu.c.linker.cygwin"
           name="%ToolName.linker.cygwin.gnu.c"
           superClass="cdt.managedbuild.tool.gnu.c.linker">
-          <envVarBuildPath 
-              pathType="buildpathLibrary"
-              variableList="LIBRARY_PATH"
-              buildPathResolver="org.eclipse.cdt.managedbuilder.gnu.cygwin.CygwinPathResolver">
-          </envVarBuildPath>
           <outputType
 			  outputs="exe"
               superClass="cdt.managedbuild.tool.gnu.c.linker.output"
@@ -2144,11 +2115,6 @@
           id="cdt.managedbuild.tool.gnu.cpp.linker.cygwin"
           name="%ToolName.linker.cygwin.gnu.cpp"
           superClass="cdt.managedbuild.tool.gnu.cpp.linker">
-          <envVarBuildPath 
-              pathType="buildpathLibrary"
-              variableList="LIBRARY_PATH"
-              buildPathResolver="org.eclipse.cdt.managedbuilder.gnu.cygwin.CygwinPathResolver">
-          </envVarBuildPath>
 		  <outputType
 			  outputs="exe"
               superClass="cdt.managedbuild.tool.gnu.cpp.linker.output"
@@ -2167,11 +2133,6 @@
           id="cdt.managedbuild.tool.gnu.c.compiler.cygwin"
           name="%ToolName.compiler.cygwin.gnu.c"
           superClass="cdt.managedbuild.tool.gnu.c.compiler">
-          <envVarBuildPath 
-              pathType="buildpathInclude"
-              variableList="CPATH,C_INCLUDE_PATH"
-              buildPathResolver="org.eclipse.cdt.managedbuilder.gnu.cygwin.CygwinPathResolver">
-          </envVarBuildPath>
           <inputType
           	  id="cdt.managedbuild.tool.gnu.c.compiler.input.cygwin"
               superClass="cdt.managedbuild.tool.gnu.c.compiler.input"
@@ -2181,11 +2142,6 @@
           id="cdt.managedbuild.tool.gnu.cpp.compiler.cygwin"
           name="%ToolName.compiler.cygwin.gnu.cpp"
           superClass="cdt.managedbuild.tool.gnu.cpp.compiler">
-          <envVarBuildPath 
-              pathType="buildpathInclude"
-              variableList="CPATH,CPLUS_INCLUDE_PATH"
-              buildPathResolver="org.eclipse.cdt.managedbuilder.gnu.cygwin.CygwinPathResolver">
-          </envVarBuildPath>
           <inputType
           	  id="cdt.managedbuild.tool.gnu.cpp.compiler.input.cygwin"
               superClass="cdt.managedbuild.tool.gnu.cpp.compiler.input"
-- 
2.42.1

